src/Client/Web/Bit.Client.Web.BlazorUI/Components/DatePicker/BitDatePicker.razor

```diff
@@ -37,7 +37,8 @@
                    @onfocusout="@HandleFocusOut"
                    @onfocus="@HandleFocus"
                    @oninput="@HandleChange"
-                   @onchange="@HandleChange" />
+                   @onchange="@HandleChange"
+                   @onblur="@HandleBlur" />
```

Comment:
as we talked about before, it needs to be handled in the `onchange` event, not in the `onblue` !

Reply:
I tried the `oninput` event but it did not work. The return value must be returned when the focus is out

Reply:
did you try the `onchange` event too?